site/content/en/docs/Start/macos.md

```diff
@@ -16,8 +16,10 @@ weight: 2
 
 If the [Brew Package Manager](https://brew.sh/) is installed, use it to download and install minikube:
 
+If installing with brew, you may need to run `brew link minikube` for installation to complete successfully.
```

Comment:
Thank you for submitting this! Could you please make the change I mentioned in this comment?

https://github.com/kubernetes/minikube/issues/5824#issuecomment-606759217

Thank you!

Reply:
OK.